docs/getting-started/installation.md

```diff
@@ -117,6 +117,12 @@ A virtual environment should solve this problem, or you can try running `python3
 The Reflex framework includes the `reflex` command line (CLI) tool. Using a virtual environment is highly recommended for a seamless experience.",
 ```
 
+```md alert warning
+# Error `Install Failed - You are missing a DLL required to run bun.exe` Windows
+Bun requires DLL run-time components of Visual C++ libraries to run on windows and fixing this issue requires installing Microsoft Visual C++ 2015 Redistributable. 
+click [here](https://aka.ms/vs/17/release/vc_redist.x64.exe) for a direct download
```

Comment:
idk about directly linking the `.exe`

Reply:
I lifted this directly from the suggestion provided in the console at the time of the error for an easy download without going to microsoft's docs. I don't mind modifying this to the download page(https://www.microsoft.com/en-us/download/details.aspx?id=53840) though